# Automatically Generated in 2017-06-12 22:33:47.453148
# MESSAGE: CONFIG_PARAM_BOOL
# Description:Config parameter information for bool param
uint8 ID = 198
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
char[30] name
uint8 value
uint8 default_value
